### DISCUSSION TITLE
I did workspace switcher with history and filter - i3rotonda

### DISCUSSION THREAD

**Discussion by micronull** [i3/i3#5564] [Show and tell]
Hey guys.

I searched and could not find a suitable switch, and ended up writing my own. 

 https://github.com/micronull/i3rotonda

Video with demonstration (use `-e 1,2` flag): https://www.youtube.com/watch?v=G93zKxe1-08

There are bugs (what can I do without them). Create issue. Thanks.

**Comment by BirdeeHub** [comment]
nevermind, I was wrong, you did a different thing than I did, all is good
